composer.json

```diff
@@ -11,23 +11,25 @@
         }
     ],
     "require": {
-        "symfony/framework-bundle": "^2.6|^3.0",
-        "symfony/security-http": "^2.6|^3.0",
-        "mixpanel/mixpanel-php": "~2.6"
+        "php": "^7.1.3",
+        "symfony/framework-bundle": "^4.3 || ^5.0",
+        "symfony/expression-language": "^4.3 || ^5.0",
+        "symfony/security-http": "^4.3 || ^5.0",
+        "mixpanel/mixpanel-php": "~2.8"
     },
     "require-dev": {
-        "phpspec/phpspec": "^3.0"
+        "phpspec/phpspec": "^5.0 || ^6.0",
+        "friendsofphp/php-cs-fixer": "^2.16"
     },
     "autoload": {
-      "psr-0": { "Gordalina\\MixpanelBundle": "" }
+      "psr-4": { "Gordalina\\MixpanelBundle\\": "" }
     },
-    "target-dir": "Gordalina/MixpanelBundle",
     "config": {
         "bin-dir": "bin"
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "2.7-dev"
+            "dev-master": "3.0-dev"
```

Comment:
What about the warning message in the README?
> Important: this bundle is developed in sync with Mixpanel's repository. If you are using Mixpanel 2.6, you need to use the ~2.6 releases of the bundle (or higher if available).

Mixpanel 3.0 has not been released yet.

Reply:
I removed this warning in the readme